tensorboard/backend/application.py

```diff
@@ -155,6 +156,8 @@ def __init__(self, logdir, plugins, multiplexer, reload_interval):
       try:
         plugin_apps = plugin.get_plugin_apps()
       except Exception as e:  # pylint: disable=broad-except
+        if isinstance(plugin, core_plugin.CorePlugin):
```

Comment:
Up to you which you want to use—the reason that I suggested `type(−) == −` instead of `isinstance(−, −)` is that it prevents tricky issues with plugins trying to extend `CorePlugin`.